go.mod

```diff
@@ -7,22 +7,25 @@ require (
 	github.com/antchfx/htmlquery v1.0.0 // indirect
 	github.com/antchfx/xmlquery v1.0.0 // indirect
 	github.com/antchfx/xpath v1.0.0 // indirect
-	github.com/aws/aws-sdk-go v1.25.1
+	github.com/denisenkom/go-mssqldb v0.0.0-20191001013358-cfbb681360f0 // indirect
```

Comment:
Where is this coming from?

Reply:
Can't be removed because `gorm` is depending on it